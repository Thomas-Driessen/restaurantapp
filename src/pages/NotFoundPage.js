import React from 'react';

const NotFoundPage = () => {
    return (
        <div style={{ padding: '3em' }}>
            <h3>Oh Sorry,</h3>
            <br></br>
            <h3>It seems that this page does not exist</h3>
        </div>
    )
}
export default NotFoundPage