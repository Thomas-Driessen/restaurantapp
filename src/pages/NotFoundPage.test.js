import React from 'react';
import { render } from '@testing-library/react';
import NotFoundPage from './NotFoundPage';
import renderer from 'react-test-renderer';

it('NotFoundPage renders correctly', () => {
  const tree = renderer
    .create(<NotFoundPage />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

test('404 Not Found', () => {
  const { getByText } = render(<NotFoundPage/>);
  const linkElement = getByText(/It seems that this page does not exist/i);
  expect(linkElement).toBeInTheDocument();
});