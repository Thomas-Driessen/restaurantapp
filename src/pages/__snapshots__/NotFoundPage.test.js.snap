// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage renders correctly 1`] = `
<div
  style={
    Object {
      "padding": "3em",
    }
  }
>
  <h3>
    Oh Sorry,
  </h3>
  <br />
  <h3>
    It seems that this page does not exist
  </h3>
</div>
`;
